tools/api-markdown-documenter/src/api-item-transforms/ApiItemTransformUtilities.ts

```diff
@@ -408,28 +429,25 @@ export function shouldItemBeIncluded(
 	apiItem: ApiItem,
 	config: ApiItemTransformationConfiguration,
 ): boolean {
-	const releaseTag = getReleaseTag(apiItem);
-	if (releaseTag === undefined || releaseTag === ReleaseTag.None) {
-		// If the item does not have a release tag, then it inherits the release scope of its ancestry.
-		const parent = getFilteredParent(apiItem);
-		if (parent === undefined) {
-			// If we encounter an item with no release tag in its ancestry, we can't make a determination as to whether
-			// or not it is intended to be included in the generated documentation suite.
-			// To be safe, log a warning but return true.
-			config.logger.warning("Encountered an API item with no release tag in ancestry.");
-			return true;
-		}
+	const releaseTag = getEffectiveReleaseLevel(apiItem);
+
+	// If the item has a release tag that is lower than the minimum release level, it should be not be included.
```

Comment:
To avoid ambiguity of what does "lower" mean in the tag hierarchy.
```suggestion
	// If the item has a release tag that is more restrictive than the minimum release level, it should not be included.
```